src/Core/System/Snippet/SnippetFileHandler.php

```diff
@@ -7,28 +7,49 @@
 use Shopware\Storefront\Storefront;
 use Symfony\Component\Finder\Finder;
 
+/**
+ * @phpstan-type Snippets array<string, string|mixed>
```

Comment:
defining the same PHPStan type twice is not necessary, you import it into another class https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases

Reply:
Didn't know that. Thanks 👍